contents/blog/new-pricing.md

```diff
@@ -0,0 +1,148 @@
+
+## How much can I save?
+
+At lower volumes, our new event pricing is 80% cheaper than before. 
+
+In real terms this means:
+
+1. Someone sending us 10 million events per month would **save $9,060 a year** if they disabled person profiles for all events.
+
+2. They would still **save $4,404 a year** if they used person profiles on 50% of their events.  
+
+We expect teams to use a combination of events with and without person profiles depending on their needs, which you can see visualized below.
+
+<br />
+
+<ProductScreenshot
+  imageLight={Pricing} 
+  imageDark={PricingDark} 
+  alt="New PostHog pricing" 
+  classes="rounded"
+/>
+
+This makes us better value than any popular (>$10M in revenue) product analytics tool.
+
+Here's how our new pricing compares to Mixpanel's:
+
+<ProductScreenshot
+  imageLight={PricingComparison} 
+  imageDark={PricingComparisonDark} 
+  alt="New PostHog pricing compared to Mixpanel" 
+  classes="rounded"
+/>
+
+And here's a more detailed breakdown of the difference at each pricing tier:
+
+| **Monthly events** | **New base event price** | **Base events + person profiles** | **Percentage diff.** |
+|--------------------|--------------------------|-----------------------------------|----------------|
+| **Up to 1m**       | Free                     | Free                              | -              |
+| **1-2m**           | $0.0000500 / event       | $0.0002480 / event                | -80%           |
+| **2-15m**          | $0.0000343 / event       | $0.0001040 / event                | -67%           |
+| **15-50m**         | $0.0000295 / event       | $0.0000655 / event                | -55%           |
+| **50-100m**        | $0.0000218 / event       | $0.0000364 / event                | -40%           |
+| **100-250m**       | $0.0000150 / event      | $0.0000187 / event               | -20%           |
+| **∞**              | $0.0000090 / event       | $0.0000100 / event                | -10%           |
+
+## Who is this for?
+
+Tracking events without person data is useful if:
+
+- You want to track your marketing website and your product, but you don't need detailed information about visitors to your marketing website.
+- You're willing to [lose some functionality](#whats-the-downside) for lower bills.
+- You want to track API or server events, especially at high volumes. 
+- You run a large e-commerce or content website with millions of users.
+- You're not tracking a high-volume event because it would be prohibitively expensive.
+- You want to use [autocapture](/docs/product-analytics/autocapture) to track events, but have turned it off to save money.
+
+We recommend you enable person profiles for logged-in users of your product and website, but disable them for unidentified users where aggregate data is sufficient. 
+
+This is how we've configured our own product and website tracking.
```

Comment:
@raquelmsmith are you ok with us saying this?

Reply:
```suggestion
For most people, we recommend using the `identified_only` [configuration option](#how-do-i-start-using-it) for person profiles. This automatically captures person profiles for logged-in users of your product and website, but doesn't capture them for unidentified users where aggregate data is sufficient. 

This is how we've configured our own product and website tracking.
```